<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>fameoucrazy</string>
  </property>
  <property name="toolTip">
   <string>fameoucrazy - a program to deal with your familys bureoucracy craziness</string>
  </property>
  <property name="iconSize">
   <size>
    <width>48</width>
    <height>48</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="toolTip">
    <string/>
   </property>
   <widget class="QTabWidget" name="central_tab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1001</width>
      <height>871</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="dasboard_tab">
     <attribute name="title">
      <string>Dash</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="client_tab">
     <attribute name="title">
      <string>Client</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="settings_tab">
     <attribute name="title">
      <string>Einstellungen</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDashboard">
    <property name="title">
     <string>Dashboard</string>
    </property>
    <addaction name="actionStartseite"/>
   </widget>
   <widget class="QMenu" name="menuKlient">
    <property name="title">
     <string>Klient</string>
    </property>
    <addaction name="actionDaten"/>
    <addaction name="actionBetreuungsumfang"/>
    <addaction name="actionBeziehungen"/>
    <addaction name="actionJournal"/>
    <addaction name="actionSchriftverkehr"/>
   </widget>
   <widget class="QMenu" name="menuProgramm">
    <property name="title">
     <string>Programm</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="actionHilfe"/>
    <addaction name="actionUpdate"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="actionGrundeinstellungen"/>
    <addaction name="actionDaten_Beziehungen"/>
   </widget>
   <addaction name="menuDashboard"/>
   <addaction name="menuKlient"/>
   <addaction name="menuProgramm"/>
  </widget>
  <action name="action_about">
   <property name="icon">
    <iconset theme="dialog-information"/>
   </property>
   <property name="text">
    <string>über</string>
   </property>
   <property name="toolTip">
    <string>über</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::AboutRole</enum>
   </property>
  </action>
  <action name="actionHilfe">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::HelpFaq"/>
   </property>
   <property name="text">
    <string>Hilfe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::SoftwareUpdateAvailable"/>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Programm beenden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::QuitRole</enum>
   </property>
  </action>
  <action name="actionDaten">
   <property name="text">
    <string>Daten</string>
   </property>
  </action>
  <action name="actionBetreuungsumfang">
   <property name="text">
    <string>Betreuungsumfang</string>
   </property>
  </action>
  <action name="actionBeziehungen">
   <property name="text">
    <string>Beziehungen</string>
   </property>
  </action>
  <action name="actionJournal">
   <property name="text">
    <string>Journal</string>
   </property>
  </action>
  <action name="actionSchriftverkehr">
   <property name="text">
    <string>Korrespundenz</string>
   </property>
  </action>
  <action name="actionGrundeinstellungen">
   <property name="text">
    <string>Grundeinstellungen</string>
   </property>
  </action>
  <action name="actionDaten_Beziehungen">
   <property name="text">
    <string>Daten &amp; Beziehungen</string>
   </property>
  </action>
  <action name="actionStartseite">
   <property name="text">
    <string>Startseite</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>about_clicked()</slot>
  <slot>help_clicked()</slot>
  <slot>upload_picture()</slot>
  <slot>on_theme_radio_changed()</slot>
 </slots>
</ui>
